README.md

```diff
@@ -3,7 +3,7 @@
 ## Install Homebrew
 
 ```bash
-/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
```

Comment:
```suggestion
/usr/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
```

Reply:
What's the point here? On my environment bash locates at /bin/bash, and /usr/bin/bash does not exist. It's unlikely that /bin is not in $PATH.

Reply:
Sorry, typo, I meant `/bin/bash`, fixed. We want to use the specific system Bash rather than a random on in the PATH.